Project.toml

```
@@ -27,6 +34,8 @@ GDAL = "1.5.1"
 GeoFormatTypes = "0.4.2"
 GeoInterface = "1"
 GeoInterfaceRecipes = "1.0"
+GeoInterfaceMakie = "0.1"
 ImageCore = "0.8, 0.9, 0.10"
+Makie = "0.20"
```

Can we make this "0.19, 0.20"? It may be a while before everyone is on 0.20
```suggestion
Makie = "0.19, 0.20"
```

---

No, because with 0.19 I get the world age errors above. https://github.com/yeesian/ArchGDAL.jl/pull/404#issuecomment-1836919842